src/dev-app/ff-perf-test/ff-perf-test.html

```diff
@@ -0,0 +1,23 @@
+<h1>With ResizeObserver</h1>
+<p>
+  how many form-fields:
+  <input type="number" [formControl]="count">
+</p>
+<p>
+  label prefix:
+  <input [formControl]="prefix">
+</p>
+<p>
+  value:
+  <input [formControl]="value">
+</p>
+<p>
+  <button (click)="hide = !hide">{{hide ? 'show' : 'hide'}} all</button>
+</p>
+
+<div [class.hidden]="hide">
```

Comment:
You can change this to `[hidden]="hide"` and get rid of the CSS for this component.

Reply:
ack, will chuck commit